backend/test_step_time.py

```diff
@@ -49,23 +51,29 @@ def test_no_movement_target_zone(self):
         self.assertEqual(target_zone, {"min": 0.0, "max": 0.0, "average": 0.0})
 
     def test_real_data_step_time_calculation(self):
-        """Test step time calculation with real data."""
-        real_force_data = [
-            (0.00000, 159.548187), 
-            (0.00093, 159.304047), 
-            (0.01296, 158.327484),  
-            (0.01620, 158.327484),  
-            (0.05370, 157.106781),  
-            (0.05972, 156.862640)   
-        ]
-        
-       
-        step_times = calculate_step_time(real_force_data, THRESHOLD)
+        """Test step time calculation with real data from the sample file."""
 
-        
-        expected_step_times = []  # Assuming no steps are calculated with this data
+        real_force_data = []
```

Comment:
Did you mean to use the load_data_from_file function here? The code seems similar except this code would solve the issue in your load_data_from_file function by checking the length of parts (my comment in target_zone_estimation.py on line 41).